webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/reports/execution/ReportsExecutionRepository.scala

```diff
@@ -87,8 +86,6 @@ trait WoReportsExecutionRepository {
  */
 class CachedReportsExecutionRepository(
     readBackend : RoReportsExecutionRepository
-  , writeBackend: WoReportsExecutionRepository
-  , findConfigs : FindExpectedReportRepository
 ) extends RoReportsExecutionRepository with WoReportsExecutionRepository with CachedRepository {
```

Comment:
You need to remove  `WoReportsExecutionRepository` here since the only method from that trait is not implemented